src/item_action.cpp

```diff
@@ -370,6 +370,11 @@ void game::item_action_menu( item_location loc )
         num++;
     }
 
+    // custom footer
+    std::string footer = string_format(_("< [<color_yellow>%s</color>] keybindings >"),
+        ctxt.get_desc("HELP_KEYBINDINGS"));
+    kmenu.footer_text = footer.substr(1, footer.length() - 3);
```

Comment:
```suggestion
    kmenu.footer_text = string_format( _( "[<color_yellow>%s</color>] keybindings" ),
        ctxt.get_desc( "HELP_KEYBINDINGS" ) );
```
Just change the original string. It will be extracted as a new string and translated on Transifex. Also code style.

Reply:
I tried the option earlier, the translation is not substituted (that's why I decided to use substr). I assume that this happens, because there is no source segment in the translation database without angular quotes. To make your version work, you can add the segment [ <color_yellow>%s</color>] assigned keys to the translation memory.

Reply:
![image](https://user-images.githubusercontent.com/29895032/212009831-ee2abbde-2867-4326-842a-03915b41f730.png)

Reply:
The addition of new strings to the translation files is done automatically via scripts after a PR is merged. There's no need to worry about translation here as long as you marked all new strings for translation (which you already does in your code).

Reply:
Qrox, can you tell me from the workflow what else I need to do? The code has been corrected, everything has been checked by AStyle, but checks are still needed.

Reply:
The remaining checks need to be approved by a maintainer. For now you don't have to do anything.